dump_offload_mgr.cpp

```diff
@@ -33,6 +33,12 @@ DumpOffloadManager::DumpOffloadManager(sdbusplus::bus::bus& bus) : _bus(bus)
 
 void DumpOffloadManager::offload()
 {
+    if (isSystemHMCManaged(_bus))
```

Comment:
we need to handle the exception right? cant crash the application

Reply:
what should be the behavior? if attribute not found should we assume it is not managed by hmc? I am crashing as i was under the assumption that the attribute must be set.

Reply:
if not set then I will assume it as not hmc managed and return?